### PR TITLE
Fix for calculating buffer sizes when calling from Python to plaidml

### PR DESCRIPTION
**Pull request by LLITCHEV**
Thanks, JB!

### CODE REVIEW

File: plaidml/core/ffi.cc [hunk 1]
```diff
@@ -369,7 +369,8 @@ uint64_t plaidml_shape_get_nbytes(  //
         total += (sizes[i] - 1) * strides[i];
       }
     }
-    return (total + 1) * (shape->type.getElementTypeBitWidth() / 8);
+    unsigned elem_bytes = (shape->type.getElementTypeBitWidth() + 7) / 8;
```
Comment (flaub) [inline plaidml/core/ffi.cc:372]: Could we use `llvm::divideCeil` here?

File: plaidml/bridge/keras/backend_test.py [hunk 2]
```diff
@@ -589,6 +587,7 @@ def testDivConstant(self, b, x, c):
             c / x,
         ]
 
+    @unittest.skip("Failing on llvm.cpu")
```
Comment (flaub) [inline plaidml/bridge/keras/backend_test.py:590]: Does this crash or just fail? If it fails, why do we skip it?

Comment (LLITCHEV) [reply plaidml/bridge/keras/backend_test.py]: Didn't know that we skip only hard crashing tests. Fixed.

File: plaidml/core/ffi.cc (hunk 1 above)
Comment (LLITCHEV) [reply plaidml/core/ffi.cc]: Yes, I think we can. Using it now.

**Pull request merged**